tests/ctst/steps/bucket-policies/utils.ts

```diff
@@ -0,0 +1,500 @@
+    // {
+    //     action: 'CreateBucket',
+    //     permissions: ['s3:CreateBucket'],
+    //     expectedResultOnAllowTest: 'BucketAlreadyOwnedByYou',
+    //     excludePermissionOnBucketObjects: true,
+    //     useWildCardBucketName: true,
+    // },
+    {
+        action: 'PutBucketAcl',
+        permissions: ['s3:PutBucketAcl'],
+        expectedResultOnAllowTest: 'MalformedXML',
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketEncryption',
+        permissions: ['s3:PutEncryptionConfiguration'],
+        expectedResultOnAllowTest: 'MalformedXML',
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketLifecycleConfiguration',
+        permissions: ['s3:PutLifecycleConfiguration'],
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketNotificationConfiguration',
+        permissions: ['s3:PutBucketNotification'],
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketPolicy',
+        permissions: ['s3:PutBucketPolicy'],
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketReplication',
+        permissions: ['s3:PutReplicationConfiguration'],
+        expectedResultOnAllowTest: 'InvalidRequest: Versioning must be \'Enabled\'',
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketVersioning',
+        permissions: ['s3:PutBucketVersioning'],
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutObjectLockConfiguration',
+        permissions: ['s3:PutBucketObjectLockConfiguration'],
+        expectedResultOnAllowTest: 'InvalidBucketState',
+    },
+    {
+        action: 'PutObject',
+        permissions: ['s3:PutObject'],
+    },
+    {
+        action: 'PutObjectLegalHold',
+        permissions: ['s3:PutObjectLegalHold'],
+        expectedResultOnAllowTest: 'MalformedXML',
+    },
+    {
+        action: 'PutObjectRetention',
+        permissions: ['s3:PutObjectRetention', 's3:BypassGovernanceRetention'],
+        needsSetup: true,
+    },
+    {
+        action: 'PutObjectTagging',
+        permissions: ['s3:PutObjectTagging'],
+    },
+    {
+        action: 'PutObjectAcl',
+        permissions: ['s3:PutObjectAcl'],
+    },
+    {
+        action: 'PutBucketCors',
+        permissions: ['s3:PutBucketCORS'],
+        expectedResultOnAllowTest: 'MissingRequestBodyError',
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketTagging',
+        permissions: ['s3:PutBucketTagging'],
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'PutBucketWebsite',
+        permissions: ['s3:PutBucketWebsite'],
+        excludePermissionOnBucketObjects: true,
+    },
+    {
+        action: 'GetBucketCors',
+        permissions: ['s3:GetBucketCORS'],
+        expectedResultOnAllowTest: 'NoSuchCORSConfiguration',
```

Comment:
this is fine but we setup to handle this in Integration https://github.com/scality/Integration/blob/development/7.10/tests/bucket-policy/utils/s3APIConfigurations.js#L210

Reply:
Several more cases in the file, probably not necessary to address but worth considering for proper testing.

Reply:
This one is returned & fine as after authz check: https://github.com/scality/cloudserver/blob/d82965ff78116998d896cf41e4cd7fbe4e21101e/lib/api/bucketGetCors.js#L60

As we only test authz, I don't think we need to implerment handlers for each API, as long as we get the right AccessDenied errors and others (or no error) in happy paths, we are safe